loops_lab.js

```diff
@@ -0,0 +1,57 @@
+
+
+
+//-------------------------------------
+let num = 5;
+while (num < 25) {
+  console.log(num);
+  num += 1;
+}
+
+for (let number = 0; number <= 10; number += 1) {
+  if (number % 2 !== 0) {
+    console.log(number + ' num is odd');
+  } else {
+    console.log(number + ' num is even');
+  }
+}
+
+let bynine = 99;
```

Comment:
This is not exactly what the question was asking but good job with printing out whether the number is divisible by 9 or not.